### ISSUE TITLE
AWX not accessible after awx-operator upgrade from 1.3.0 to 1.4.0

### ISSUE THREAD

**Issue by A-Kumpf** [ansible/tower-operator#1310]
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx-operator/issues) for duplicates.
- [X] I understand that the AWX Operator is open source software provided for free and that I might not receive a timely response.

### Bug Summary

Hi there

After upgrading from 1.3.0 to 1.4.0, login is not possible.

Downgrade to awx-operator 1.3.0 worked with no problem.

Greets

### AWX Operator version

1.4.0

### AWX version

21.14.0

### Kubernetes platform

kubernetes

### Kubernetes/Platform version

1.24.10

### Modifications

yes

### Steps to reproduce

Upgrade from 1.3.0 to 1.4.0

### Expected results

a functional AWX

### Actual results

no login possible, white screen in browser

### Additional information

Changed awx-operator/config/manager/kustomization.yaml to not use latest but the actual awx-operator version to deploy.

### Operator Logs

task container log:

2023-03-29 14:56:28,709 ERROR [a2443c666eb648149d671fbc6e58ceb9] awx {'csp-report': {'blocked-uri': 'eval', 'document-uri': 'https://xxxxxxxxxxxxxx', 'effective-directive': 'script-src', 'original-policy': "default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'unsafe-inline'; script-src 'self' 'nonce-psiiHQKI3e9sxg54iho7bmzgEArfDHb/bvk3vw6Mjd8=' https://*.pendo.io; img-src 'self' https://*.pendo.io data:; worker-src 'self' blob:", 'referrer': '', 'status-code': 200, 'violated-directive': 'script-src'}}
10.42.2.156 - - [29/Mar/2023:14:56:28 +0000] "POST /csp-violation/ HTTP/1.1" 200 4 "https://xxxxxxxxxxx" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0" "ip.ip.ip.ip"
[pid: 35|app: 0|req: 36/79] 10.42.2.156 () {70 vars in 1101 bytes} [Wed Mar 29 14:56:28 2023] POST /csp-violation/ => generated 4 bytes in 36 msecs (HTTP/1.1 200) 7 headers in 266 bytes (1 switches on core 0)
2023-03-29 14:56:28,753 ERROR [08ae86e676f94a958b734d46384a6b95] awx {'csp-report': {'blocked-uri': 'eval', 'document-uri': 'https://xxxxxxxxxxxxxxxxxxx', 'effective-directive': 'script-src', 'original-policy': "default-src 'self'; connect-src 'self' ws: wss:; style-src 'self' 'unsafe-inline'; script-src 'self' 'nonce-psiiHQKI3e9sxg54iho7bmzgEArfDHb/bvk3vw6Mjd8=' https://*.pendo.io; img-src 'self' https://*.pendo.io data:; worker-src 'self' blob:", 'referrer': '', 'status-code': 200, 'violated-directive': 'script-src'}}
10.42.2.156 - - [29/Mar/2023:14:56:28 +0000] "POST /csp-violation/ HTTP/1.1" 200 4 "https://paratower.ethz.ch/" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:109.0) Gecko/20100101 Firefox/111.0" "ip.ip.ip.ip"

**Comment by A-Kumpf** [comment]
Hoi there

Since pulling the newest version (today, 31/03/23) deployment worked and I was able to log in again (no white screen anymore).
About shows AWX version 21.14.0 and awx-operator version 1.4.0 was pulled from quay.io.
Running a test template was successful as well.
Keep investigating ...

Regards
Andi

**Comment by A-Kumpf** [comment]
Correction

It's not working again because of Content Security Policy (csp) violation:
2023-04-03 05:52:32,101 ERROR [beeaba33a3534482bd2a6fc376880322] awx {'csp-report': {'blocked-uri': 'eval', 'document-uri': .............................
'self' 'unsafe-inline'; script-src 'self' 'nonce-14AiV0x2h8HZ9vDi/STxMVrWI/yzjJZqgFDn+qLc8RE=' https://*.pendo.io; img-src 'self' https://*.pendo.io data:; worker-src 'self' blob:", 'referrer': '', 'status-code': 200, 'violated-directive': 'script-src'}}
10.42.2.156 - - [03/Apr/2023:05:52:32 +0000] "POST /csp-violation/ HTTP/1.1" 200 4 "https

Regards
Andi

**Comment by fosterseth** [comment]
@A-Kumpf were you able to resolve this?

**Comment by A-Kumpf** [comment]
> @A-Kumpf were you able to resolve this?

Nope, my issue seems to be present in some environments and in some not. Even using different browsers is not consistently error producing.

How can I completely turn off CSP? Thus, nginx used in the web container has no (custom) csp configuration.

Thx anyway for the reply.

**Comment by djyasin** [comment]
Hi @A-Kumpf,
Have you reached out to the mailing list (https://github.com/ansible/awx/#get-involved)  to see if any other community members have encountered this issue?
